src/query/sql/src/planner/semantic/type_check.rs

```diff
@@ -3509,6 +3512,7 @@ impl<'a> TypeChecker<'a> {
                     arg_x,
                 ]))
             }
+            ("iff", args) => Some(self.resolve_function(span, "if", vec![], args)),
```

Comment:
better to registe alias function name in this file `src/query/functions/src/scalars/control.rs`
```rust
registry.register_aliases("if", &["iff"]);
```

Reply:
evaluator judge `if name == "if"` in multi times, so we don't introduce function alias in expression level.